OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -177,7 +181,8 @@ public MovementType CurrentMovementTypes
 		INotifyMoving[] notifyMoving;
 		INotifyFinishedMoving[] notifyFinishedMoving;
 		IWrapMove[] moveWrappers;
-		bool requireForceMove;
+		public bool RequireForceMove;
```

Comment:
I don't remember the exact technical or style reason, but I do remember that we generally prefer the approach of `IsBlocking` below (`{ get; private set; }`) when all updates happen privately and external places are only supposed to read it.
Same for `Immovable` below.

Reply:
Actually, after the third commit this doesn't need to be public at all anymore, so reverted that in the third commit. Fixed for (Is)Immobile.